src/autorenderloop.js

```diff
@@ -143,9 +143,16 @@ olcs.AutoRenderLoop.prototype.enable = function() {
     that.notifyRepaintRequired();
   };
 
-  // Listen for changes on the layer group
-  this.ol3d.getOlMap().getLayerGroup().on('change',
+  // Listen for changes on the main layer group
+  this.ol3d.getOlMap().getLayerGroup().on('change:layers',
       this._boundNotifyRepaintRequired);
+
+  this.ol3d.getOlMap().getLayers().forEach((layer) => {
+    // Listen for changes on layer group
+    layer.on('change', that._boundNotifyRepaintRequired);
+    // Listen for property changes on simple layer (opacity, visibility, ...)
+    layer.on('propertychange', that._boundNotifyRepaintRequired);
+  });
```

Comment:
yes, this could probably be relaxed because we don't have this behavior with the ol.layer.Layer (changed() is called even if the layer is not visible)